src/Test.FunctionalTests.Sarif/v2/ConverterTestData/FxCop/FxCopExceptions.xml.sarif

```diff
@@ -28,6 +23,11 @@
           }
         }
       ],
+      "tool": {
```

Comment:
>tool [](start = 7, length = 4)

This data move happens now when we refresh our test files via automated scripts, so I've gone ahead and checked the change in.